app/containers/Scatter/saga.js

```diff
@@ -93,7 +93,7 @@ function* getAccountDetail(name) {
 function* refreshEosAccountData() {
   const accountName = yield select(makeSelectEosAccount());
   try {
-    if (accountName && accountName !== 'Attach an Account') {
+    if (accountName && accountName !== '') {
```

Comment:
Can you just check for `accountName` truthy, rather than checking for both the truthy and the text being empty?

Reply:
Good point, was just being lazy